src/extension.ts

```diff
@@ -208,10 +208,11 @@ class ClientWorkspace {
     };
 
     const pattern = this.config.multiProjectEnabled
-      ? `${this.folder.uri.path}/**`
+      ? `**${this.folder.uri.path}/**`
       : undefined;
+
     const collectionName = this.config.multiProjectEnabled
-      ? `rust ${this.folder.uri.toString()}`
+      ? `rust ${this.folder.uri.path}`
```

Comment:
Is it the same instance as https://github.com/rust-lang/rls-vscode/pull/759#discussion_r409780233?

Reply:
yes.. :sweat_smile:

Reply:
The only place where it is necessary is for the pattern

Reply:
Thanks for responding so quickly and making me understand the changes better!